yarn.lock

```diff
@@ -26730,7 +26626,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"node-gyp@npm:^10.0.0, node-gyp@npm:latest":
```

Comment:
@montelaidev This (downgrading `node-gyp@npm:latest` to v9 instead of v10) should be reverted to resolve failing `deps-audit` check.